README.md

```diff
@@ -1,5 +1,89 @@
+
+Because this uses the scaffold you still need to run `npm install` after forking and cloning the repository. You do not need to run `npm start` until waves two and three, because until then your project code will not be a web application.
+
+To run your tests you should use the command `npm test`.
+
+#### Using the REPL
+If you want to interact with your game in the terminal via a REPL, you'll need to do things in a specific way (due to the scaffold's use of webpack).
+
+To access the REPL you should use the command `npm run repl` in your project directory. Once in the REPL, you'll need to use the following syntax to import your file (this example assumes it was named `src/game.js`):
+
+```javascript
+var Game = require('game').default;
+var game = new Game();
+```
+
+NOTE: This only works if your `game.js` file has an `export default` line in it, which it should (it'll be necessary for the Backbone conversion).
+
+## Requirements
+This project is divided into three waves that will be completed over the next two weeks. The majority of the first week will be dedicated to wave one, with the second and third waves taking place the week after.
+
+### Wave One
+The primary learning goals in this wave are writing unit tests for JavaScript with Jasmine, and program/code design. What we mean by program design is that the specific approach to implementing your Tic-Tac-Toe game's logic will be up to you.
+
+Below we have laid out the requirements for your implementation in the form of user stories. Because user stories do not specify an implementation or interface (i.e. they do not say you must make a function named `highestScoreFrom`), there's a lot of flexibility for what approach you take.
+
+#### Program Design
+The overarching goal with the program design requirement is for you to spend more time at the beginning of the project thinking about what functionality the application needs, how that can be split up into individual functions, and what the inputs and outputs of those functions are.
+
+This might seem like an unwarranted distraction from diving right into the code when you're under such time constraints. But a great deal of time can be saved by avoiding costly refactors if it turns out that your initial approach won't allow you to achieve certain requirements. The more complex the application's logic is, the more difficult and time-consuming it will be to rework that logic after it has been written.
+
+#### User Stories
+- As a user of the Tic-Tac-Toe game I CAN:
+  - Create a new game:
+    - It will have an empty board of nine cells
+    - It will start with the X player having the first move
+  - Specify up to two player names
+  - Get the current outcome of the game (X win, O win, tie, or undecided)
+  - (OPTIONAL) View a printed version of the current game board for debugging purposes
+- As the 'active' player in a game I CAN:
```

Comment:
This is a little bit confusing about having a separate "player" object, or just having this logic in a part of the game. Is there a way we can help clarify this part?